performance-storage-service/pss_project/settings/local.py

```diff
@@ -1,7 +1,7 @@
 from .base import *
 
 DEBUG = True
-ALLOWED_HOSTS = ['0.0.0.0', 'kubernetes.docker.internal', 'localhost', '127.0.0.1', 'host.docker.internal']
+ALLOWED_HOSTS = ['0.0.0.0', 'kubernetes.docker.internal', 'localhost', '127.0.0.1', 'host.docker.internal', 'smee.io']
```

Comment:
Do we still need 'smee.io'?

Reply:
Probably not but it could be a good way to test locally so I don't think there is much harm in keeping it